### ISSUE TITLE
Install via cli throws error

### ISSUE THREAD

**Issue by jorenvanhee** [engram-design/CPNav#53]
### Description

The plugin throws an error if you install it through the cli.

```
$ ./craft install/plugin cp-nav
*** installing cp-nav
    > create table {{%cpnav_layout}} ... done (time: 0.018s)
    > create table {{%cpnav_navigation}} ... done (time: 0.023s)
    > add foreign key cpnav_navigation_layoutId_fk: {{%cpnav_navigation}} (layoutId) references {{%cpnav_layout}} (id) ... done (time: 0.023s)
*** failed to install cp-nav: Calling unknown method: craft\console\Request::getUrl()
```

### Steps to reproduce

1. `composer require verbb/cp-nav`
2. `./craft install/plugin cp-nav`

### Additional info

- Plugin version: 2.0.8
- Craft version: 3.0.28

**Comment by nicolasbinet** [comment]
same here.

verbb/cp-nav/src/CpNav.php:80 refers to Craft::$app->getRequest()->getUrl() which is not available on CLI

**Comment by engram-design** [comment]
Fixed in [2.0.9](https://github.com/verbb/cp-nav/releases/tag/2.0.9)
